modules/global-classes/global-classes-repository.php

```diff
@@ -67,6 +72,12 @@ public function put( array $items, array $order ) {
 			throw new \Exception( 'Failed to update global classes' );
 		}
 
+		$deleted_class_ids = $this->get_deleted_class_ids( $current_value, $updated_value );
+
+		if ( ! empty( $deleted_class_ids ) ) {
+			Elements_Classes::iterate_data( $this->handle_deleted_classes( $deleted_class_ids ) );
```

Comment:
```suggestion
			$this->handle_deleted_classes( $deleted_class_ids );
```

I think the iteration should happen in the lower level, to keep the same level of abstraction here